object-detection/object_detection/modules/model.py

```diff
@@ -10,6 +10,9 @@
 from core.path import GSPath
 from torch import nn
 
+# pylint: disable=protected-access
+torch.hub._validate_not_a_forked_repo = lambda a, b, c: True
+
```

Comment:
```suggestion
from torch import nn

# workaround to avoid 403 HTTP errors from torch.hub, 
# see https://github.com/pytorch/pytorch/issues/61755
# pylint: disable=protected-access
torch.hub._validate_not_a_forked_repo = lambda a, b, c: True
```